indexd/index/blueprint.py

```diff
@@ -94,6 +93,9 @@ def get_index():
     metadata = flask.request.args.getlist('metadata')
     metadata = {k: v for k, v in map(lambda x: x.split(':', 1), metadata)}
 
+    urls_metadata = flask.request.args.get('urls_metadata')
+    urls_metadata = urls_metadata and json.loads(urls_metadata)
```

Comment:
The urls_metadata is passed as a json string in the url param.

Reply:
catch if it's an invalid json